src/Api/Controllers/AccountsController.cs

```diff
@@ -467,14 +467,19 @@ public async Task<PaymentResponseModel> PostPremium(PremiumRequestModel model)
             {
                 license = await ApiHelpers.ReadJsonFileFromBody<UserLicense>(HttpContext, model.License);
             }
+            else if (!valid && !_globalSettings.SelfHosted)
+            {
+                throw new BadRequestException("Country is required.");
```

Comment:
How do we know this is why it is not valid?

Reply:
Magic, duh 
![image](https://user-images.githubusercontent.com/3904944/85032330-b2e26100-b14d-11ea-9d3b-7a6694179766.png)